README.md

```diff
@@ -1213,6 +1213,94 @@ Here is a complete example for copying bundles from `prod` to `integration`.
    The `--map-version` option adds a specific duration to the version of each
    copied file and bundle. Run `python scripts/copy_bundles --help` for details.
 
+## 7.4 Debugging running lambdas via Pycharm
+
+It's possible to connect a remote debugger to a running lambda function.
+
+1. In PyCharm select `Run > Edit Configuration`, then The **+** to add a new
+   **Python Debug Server**.
+
+   Enter your IP address under **IDE host name**. If you don't have a static IP,
+   leave the **host name** as `localhost` but install https://ngrok.com/. We
```

Comment:
Also consider using embedding link and linking directly to [the download page](https://ngrok.com/download)

Reply:
You have to make an account and do some other steps aside from downloading ngrok in order to use it. I think it's easier to understand how it works if you start on their landing page.